### PR TITLE
Fix part of #2833:Added Named Arguments in DragDropTestActivityTest

### PR DESCRIPTION
**Pull request by Naman-1234**
## Explanation
 Fixes Part of #2833 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by FareesHussain** [comment]
@Naman-1234 there are some [lint errors](https://github.com/oppia/oppia-android/pull/2867/checks?check_run_id=2069368047)

re-run the `scripts/setup.sh` to fix these before pushing the code

**Comment by Naman-1234** [comment]
@FareesHussain  Sir There is an error coming while Pushing. Could you please help me with this.

<p align="center">
Error
</p>

**Comment by FareesHussain** [comment]
@Naman-1234 use line breaks for this and before committing the code you can run `scripts/ktlint_lint_check.sh` to check if the current changes are correct.

**Comment by Naman-1234** [comment]
@FareesHussain  Thanks Sir It helped a lot, I think Errors have been fixed now.

**Review by FareesHussain — CHANGES_REQUESTED** [review]
The argument name should be the same as the name of the recieveing argument.

Reference - https://kotlinlang.org/docs/functions.html#named-arguments


Have look at these [errors](https://github.com/oppia/oppia-android/pull/2867/checks?check_run_id=2070191634) 
```bash
public open fun withId(id: Int): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
public open fun withId(integerMatcher: Matcher<Int!>!): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (106, 14): None of the following functions can be called with the arguments supplied: 
public open fun withId(id: Int): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
public open fun withId(integerMatcher: Matcher<Int!>!): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (132, 14): None of the following functions can be called with the arguments supplied: 
public open fun withId(id: Int): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
public open fun withId(integerMatcher: Matcher<Int!>!): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (133, 14): None of the following functions can be called with the arguments supplied: 
public open fun withId(id: Int): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
public open fun withId(integerMatcher: Matcher<Int!>!): Matcher<View!>! defined in androidx.test.espresso.matcher.ViewMatchers
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (155, 7): Named arguments are not allowed for non-Kotlin functions
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (155, 7): Cannot find a parameter with this name: recyclerViewId
e: /home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt: (156, 5): No value passed for parameter 'p0'


```

File: app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ class DragDropTestActivityTest {
       scenario.onActivity { activity ->
         attachDragDropToActivity(activity)
       }
-      onView(withId(R.id.drag_drop_recycler_view)).perform(
+      onView(withId(recyclerViewId = R.id.drag_drop_recycler_view)).perform(
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt]: Do not add named arguments to `withId`.
Update this to `onView(withId(R.id.drag_drop_recycler_view))`

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt]: This is the reason why tests are failing too.

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt]: @Naman-1234 PTAL

Comment (Naman-1234) [reply app/src/sharedTest/java/org/oppia/android/app/testing/DragDropTestActivityTest.kt]: Ok Sir Working on it.

**Comment by Naman-1234** [comment]
@rt4914  Sir, Errors has Been removed now, Thanks for guiding me.

**Review by anandwana001 — APPROVED** [review]
LGTM

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**